crates/xtask/src/install.rs

```diff
@@ -31,3 +28,22 @@ pub fn install<P: AsRef<Path>>(config: &Config<'_>, game_dir: P) -> Result<()> {
 
     Ok(())
 }
+
+fn recursive_copy<P1: AsRef<Path>, P2: AsRef<Path>>(
+    source: &P1,
+    dest: &P2,
+    sh: &Shell,
+) -> Result<()> {
```

Comment:
I like the `impl` syntax, maybe it could be used here to make it less verbose
```suggestion
fn recursive_copy(
    source: impl AsRef<Path>,
    dest: impl AsRef<Path>,
    sh: &Shell,
) -> Result<()> {
```

Reply:
Since you prefer impl syntax, I took that as the project style and converted most of the generic types to impl syntax.